integration-test/src/main/java/com/sequenceiq/it/cloudbreak/actor/CloudbreakUserCache.java

```diff
@@ -39,6 +40,8 @@
     @Value("${integrationtest.ums.deploymentKey:}")
     private String realUmsUserDeployment;
 
+    private boolean useUmsStore;
```

Comment:
i would not store this flag here, this cache is related to users, this flag is a global test config which decides we are using ums or not